core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -209,14 +209,14 @@
     max-width: 560px;
     /* NOTE TO DEVELOPERS: If min-width is changed, max-width in media query
        below should be changed to match. */
-    min-width: 360px;
+    min-width: 447px;
     width: 100%;
     z-index: 3;
   }
 
   /* Some mobile devices have CSS width below 360px, use a responsive min-width
      when under 360px to prevent pushing things offscreen. */
-  @media(max-width: 360px) {
```

Comment:
Just wondering, I assume you're adding 87px here but what effect does that have?

Also, you should update the above comment so its clear and matches your new max-width value.

Reply:
I think we don't need to add 87px here. 360px works fine as we are just fixing min-width of the div to 100vw when max-width is 360px for some mobile devices.
I have reverted the changes @AllanYangZhou .
Let me know if I am wrong here.
Thanks